runtime/sema/errors.go

```diff
@@ -2703,6 +2703,15 @@ func (e *InvalidPathDomainError) Error() string {
 	return "invalid path domain"
```

Comment:
Good idea to add the actual identifier in `InvalidPathIdentifierError.Error`, maybe do the same here